app/src/main/kotlin/no/nav/tms/mikrofrontend.selector/DisableSink.kt

```diff
@@ -28,9 +29,21 @@ class DisableSink(
     }
 
     override fun onPacket(packet: JsonMessage, context: MessageContext) {
-        log.info { "mottok disablemelding for ${packet.microfrontendId}" }
-        personRepository.disableMicrofrontend(packet)
-        DisableMessage.countVersion(packet)
+        traceMicrofrontend(id = packet.microfrontendId, extra = packet.traceInfo("disable")) {
+            try {
+                log.info { "disablemelding mottat fra ${packet.initiatedBy}" }
+                personRepository.disableMicrofrontend(packet)
+                DisableMessage.countVersion(packet)
+            } catch (e: Exception) {
+                log.error { "Feil i behandling av enablemelding" }
+                secureLog.error { """
+                    Feil i behandling av enablemelding for person med ident ${packet.ident}
+                    ${e.stackTrace}
```

Comment:
Funker ikke som tenkt. Bør være `secureLog.error(e) { "Feil i behandling av enablemelding for person med ident ${packet.ident}" }`